setup.cfg

```diff
@@ -70,10 +70,12 @@ tests =
     pytest-cov>=2.5.1
     pre-commit
     matplotlib>=3.4.2
+    IPython
     uvtools @ git+git://github.com/HERA-Team/uvtools.git
     hera_cal @ git+git://github.com/hera-team/hera_cal
     healvis @ git+git://github.com/rasg-affiliates/healvis
     bda @ git+git://github.com/HERA-Team/baseline_dependent_averaging.git
+    vis_cpu @ git+git://github.com/HERA-Team/vis_cpu.git
```

Comment:
We're trying to release hera_sim as a PyPI package, and as such it can't have git-url requirements like this (I've made issues on all the other repos above this to this effect as well). We should just release a new vis_cpu version so that this branch works with the PyPI version of vis_cpu.

Reply:
Thanks, this was added in this PR just to get the tests to work properly. Now the vis_cpu version has been bumped I've removed it.